migration/src/m0000030_create_sbom.rs

```diff
@@ -26,7 +26,14 @@ impl MigrationTrait for Migration {
                     .if_not_exists()
                     .col(ColumnDef::new(Sbom::SbomId).uuid().not_null())
                     .col(ColumnDef::new(Sbom::NodeId).string().not_null())
-                    .col(ColumnDef::new(Sbom::Location).string().not_null())
```

Comment:
I don't understand why any changes to existing migrations are required. Kinda defeats the purpose of migrations, no?

Reply:
The change is, that it allows the us of the (now) deprecated `Sbom::Location` variant. That changes that line, but that only affects the compilation/check phase.

Reply:
I now understand why "allowing the use" is required: because we have migrations that import other migrations. IMHO, this is a confusing practice because each migration should be self-contained. Any `DeriveIden` enums should be private and only contain variants relevant to the migration in which they're defined.

Reply:
That would require us to re-define identifiers locally. Which I believe contradicts the whole idea of such identifier definitions. But if that's what it takes to get the PR merged …